<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addSubscriberDialog</class>
 <widget class="QDialog" name="addSubscriberDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加订阅信息</string>
  </property>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>22</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>56</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>58</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>117</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>117</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>157</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>杂志单位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="magazineBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>154</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>88</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>190</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>订阅数：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantityBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>186</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>217</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>截止日期：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="deadlineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>216</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="manButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>87</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="womanButton">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>87</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
